internal/data-ingest-cli/commands/netflow/README.md

```diff
@@ -0,0 +1,61 @@
+# Set up debug environment with Netflow receiver
+
+The primary purpose of setting up a debug environment with the NetFlow receiver is to inspect the behavior of the NetFlow receiver and the entire collector while feeding it with data using the [nflow-generator](https://github.com/nerdalert/nflow-generator) application. This debug environment setup substitutes the standard setup using the CLI debug tool to emit NetFlow data.
+
+A known limitation of the `nflow-generator` tool is that it can only send `netFlow v5` data and hence misses the `netflow v9`, `netflow IPFIX` and `sflow v5` formats, which are currently supported by the [OTel Netflow receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/netflowreceiver).
+
+## Prerequisities
+
+- Podman v5
+
+## Set up
+
+1. Retrieve you local IP address
+
+Depending on your OS, you should retrieve your local IP address so Netflow generator running in a Podman container can send data to your collector running locally. This is convenient because you can run your collector via an IDE and therefore have more debug options available.
+
+On macOS or Linux, you can display your network interfaces via:
+
+```shell
+ifconfig
+```
+
+You should search for `en0` (MacOS) or `eth0` (Linux), but it can differ on every system.
```

Comment:
```suggestion
You should search for `en0` (macOS) or `enp0` (Linux), but it can differ on every system.
```
It will be things like `enp0s25` on most Linux systems now.